universal-application-tool-0.0.1/test/repository/ProgramRepositoryTest.java

```diff
@@ -142,4 +142,45 @@ public void returnsAllAdmins() throws ProgramNotFoundException {
     Program newDraft = repo.createOrUpdateDraft(withAdmins);
     assertThat(repo.getProgramAdministrators(newDraft.id)).containsExactly(admin);
   }
+
+  // @Test
```

Comment:
this is a big bummer and a matter of me being pressed for time and unable to run unit tests locally. a few options:

a) leave it commented out with a TODO to test it after i get my unit tests working, next week perhaps
b) pair on it tomorrow, hopefully get some decent tests in in time to do a prod release
c) delay merging until i get my unit tests figured out

given the urgency of the request i'm leaning toward (a) but i don't feel great about it

Reply:
I can try to help with B tomorrow.  I feel like these situations are exactly why we shouldn't do A unless we have strong confidence it's working as expected otherwise.

Reply:
Do you think the browser tests cover this sufficiently?

Reply:
I do think there is pretty decent browser test coverage. Multiple tests exercise this view to confirm behavior in other parts of the system.